linter/linter.js

```diff
@@ -14,60 +12,599 @@ debugger;
 console.log("world!");
 `;
 
-const ast = convertAST(
-  parse(input, {
-    sourceType: "module",
-    loc: true,
-    sourceFilename: "test.js",
-    tokens: true,
-    ranges: true,
-  })
-);
-
-console.log(ast);
-
-const sourceCode = new SourceCode(input, ast);
+const ast = {
```

Comment:
I copied this output from astexplorer.net using the `espree` parser with `loc`, `range`, `tokens`, `comments`, `jsx` options enabled.